sdk/src/test/java/com/mapbox/maps/MapAttributeParserTest.kt

```diff
@@ -144,6 +144,34 @@ class MapAttributeParserTest {
     )
   }
 
+  @Test
+  fun mapAttributeParserOptimizeForTerrainEnabledTrue() {
+    every {
+      typedArray.getBoolean(
+        R.styleable.mapbox_MapView_mapbox_optimizeForTerrainEnabled,
+        true
+      )
+    } returns true
+    assertEquals(
```

Comment:
You could just use `assertTrue` or `assertFalse` instead.

Reply:
Valid point, thanks
But I supported previous style and assertTrue requires non null value. But `optimizeForTerrain` from `MapOptions` can be null. So as for me `assertEquals ` more suitable here